### ISSUE TITLE
Show deployment event logs to check if a deployment with ECS controller failed.

### ISSUE THREAD

**Issue by felipem1210** [aws-actions/amazon-ecs-deploy-task-definition#447]
I tested what you had on the "Troubleshooting" section of the README and I thought that I was going to see the logs showed on ECS deployment events. 

Maybe is not the intention, but the action is not being very useful on my use case. In my use case, the developers are constantly creating deployments and when it fails they are not all capable to see and determine the root cause of failure.

I want to be able to output the latest events logs and give some guidance so the devs can troubleshoot on their own.

**Comment by joshuabalduff** [comment]
Yeah, I also noticed that it takes like 30 minutes for a service to fail which is crazy, this is not really the actions fault but wish there was a way to end the deploy after a few failures.

**Comment by vgouv** [comment]
I was about to search or create a solution for this.
In our case things are worst, as for production deploymentis we have the circuit breaker enabled, along with rollback to previous state. Then it can happen that the github action ends with sucess code, while the deployment is rolled back (and successful).
I think this will need a new PR apart from https://github.com/aws-actions/amazon-ecs-deploy-task-definition/pull/454, but it can provide a good starting point.
